orchagent/vxlanorch.cpp

```diff
@@ -429,17 +474,17 @@ bool VxlanTunnel::createTunnel(MAP_T encap, MAP_T decap, uint8_t encap_ttl)
     return true;
 }
 
-sai_object_id_t VxlanTunnel::addEncapMapperEntry(sai_object_id_t obj, uint32_t vni)
+sai_object_id_t VxlanTunnel::addEncapMapperEntry(sai_object_id_t obj, uint32_t vni, tunnel_map_type_t type)
 {
-    const auto encap_id = getEncapMapId();
-    const auto map_t = tunnel_map_.first;
+    const auto encap_id = getEncapMapId(type);
+    const auto map_t = tunnel_map_type(type,true);
     return create_tunnel_map_entry(map_t, encap_id, vni, 0, obj, true);
 }
 
-sai_object_id_t VxlanTunnel::addDecapMapperEntry(sai_object_id_t obj, uint32_t vni)
+sai_object_id_t VxlanTunnel::addDecapMapperEntry(sai_object_id_t obj, uint32_t vni, tunnel_map_type_t type)
 {
-    const auto decap_id = getDecapMapId();
-    const auto map_t = tunnel_map_.second;
+    const auto decap_id = getDecapMapId(type);
+    const auto map_t = tunnel_map_type(type,false);
```

Comment:
miss to change the function `tunnel_map_type(type, true)` ?? In current source, this function is declared as  `tunnel_map_type (MAP_T map_t)`

Reply:
Done